activiti-cloud-notifications-graphql-service/services/jpa-query/src/main/java/org/activiti/cloud/services/notifications/graphql/jpa/query/ActivitiGraphQLSchemaAutoConfiguration.java

```diff
@@ -44,9 +42,7 @@
 public class ActivitiGraphQLSchemaAutoConfiguration {
 
     @Bean
-    GraphQLJPASchemaBuilderCustomizer graphQLJPASchemaBuilderCustomizer(
-        @Value("${activiti.cloud.graphql.jpa-query.date-format:yyyy-MM-dd'T'HH:mm:ss.SSSX}") String dateFormatString
-    ) {
```

Comment:
Was this customization related to to some bug that was fixed now?

Reply:
The Hibernate 6.3.x had changed the date/time data type semantics to use specialized Java date/time types based on the JPA `@Temporal` attribute qualifier, so this custom override is no longer needed since I updated Query entities attributes to use `@Temporal(TIMESTAMP)` which automatically provides coercion to ISO Timestamp date format.